charts/ndb-operator/templates/deployment-ndb-operator-controller-manager.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: {{ include "ndb-operator.fullname" . }}-controller-manager
+  namespace: {{.Release.Namespace}}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      control-plane: {{ include "ndb-operator.fullname" . }}-controller-manager
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        control-plane: {{ include "ndb-operator.fullname" . }}-controller-manager
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
```

Comment:
Line 23-28 were copied over from the older main branch, as they were not included in the manifests generated using the "make deploy" command.